addons/portal/static/tests/tours/skip_to_content.js

```diff
@@ -13,7 +13,7 @@ registry.category("web_tour.tours").add("skip_to_content", {
             content: "Check if we have been redirected to #wrap",
             trigger: "body",
             run: () => {
-                if (!window.location.href.endsWith("/#wrap")) {
+                if (!window.location.href.endsWith("#wrap")) {
```

Comment:
How can we be sure `#wrap` will be always at the end of the string?

Reply:
Hello @xavierbol, Skip to content is an anchor link with href  ```#wrap```. So when clicked, it appends itself to the ```url```. Or maybe we could use ```String.includes(searchString)```. Am I missing something ? What do you think ?